src/SixLabors.Fonts/AppliedFontStyle.cs

```diff
@@ -1,15 +1,47 @@
-﻿// Copyright (c) Six Labors and contributors.
+// Copyright (c) Six Labors and contributors.
 // Licensed under the Apache License, Version 2.0.
 
+using System.Collections.Generic;
+using System.Numerics;
+
 namespace SixLabors.Fonts
 {
     internal struct AppliedFontStyle
     {
-        public IFontInstance Font;
+        public IEnumerable<IFontInstance> FallbackFonts;
+        public IFontInstance MainFont;
         public float PointSize;
         public float TabWidth;
         public int Start;
         public int End;
         public bool ApplyKerning;
+
+        public GlyphInstance GetGlyph(int codePoint)
+        {
+            GlyphInstance glyph = this.MainFont.GetGlyph(codePoint);
+            if (glyph.Fallback)
+            {
+                foreach (var f in this.FallbackFonts)
```

Comment:
How do we ensure here that re-enumerating the enumerable here is not costly?

Reply:
its only ever called once.

Reply:
skip that wrong file

Reply:
sorted... now using an Array not IEnumerable